_posts/2024-04-02-Multilingual-Search.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: post
+title:  "Multilingual search with OpenSearch"
+authors:
+- roigam
+date: 2024-04-01 13:30:00 -0600
+categories:
+  - technical-post
+meta_keywords: semantic search engine, multilingual search, keyword and natural language search, search relevance, embeddings
+meta_description: Learn how to use OpenSearch to conduct multilingual search with models that support multilingual embeddings.
+---
```

Comment:
Please update the meta with the following:

**meta_keywords:** OpenSearch semantic search, multilingual search, multilingual embedding models, natural language and search

**meta_description:**  Learn how OpenSearch semantic search can help you understand the intent of a search query and conduct multilingual search with models that support multilingual embeddings.

Reply:
done